src/bessel.jl

```diff
@@ -613,3 +614,37 @@ hankelh1x(nu, z) = besselhx(nu, 1, z)
 Scaled Bessel function of the third kind of order `nu`, ``H^{(2)}_\\nu(x) e^{x i}``.
 """
 hankelh2x(nu, z) = besselhx(nu, 2, z)
+
+function _besselhs(nu::Float64, k::Int32, z::Complex{Float64}, N::Int32)
+    ai1, ai2 = Array{Float64}(N), Array{Float64}(N)
+    ae1, ae2 = Ref{Int32}(), Ref{Int32}()
+
+    ccall((:zbesh_,openspecfun), Cvoid,
+           (Ref{Float64}, Ref{Float64}, Ref{Float64}, Ref{Int32}, Ref{Int32}, Ref{Int32},
+            Ptr{Float64}, Ptr{Float64}, Ref{Int32}, Ref{Int32}),
+            real(z), imag(z), nu, Int32(1), k, N,
+            ai1, ai2, ae1, ae2)
+
+    if ae2[] == 0
+        return (N == 1 ? complex(ai1[1], ai2[1]) : complex.(ai1, ai2))
+    elseif ae2[] == 3
+        warn("besselh: loss of precision")
```

Comment:
Is this something that we do in other places? I didn't think we typically gave warnings like this. If we do, this should be `Compat.@warn "..."` for compatibility with Julia 0.7.

Reply:
Personally I prefer knowing when the loss of precision begins, but I removed this as it is not done with the other functions.